src/component/SettingForm/index.jsx

```diff
@@ -0,0 +1,68 @@
+import { PERIODS } from '../../services/timer';
+
+const SettingForm = () => {
+  const [mintWork, setMintWork] = useState(PERIODS.work.mins);
+  const [secWork, setSecWork] = useState(PERIODS.work.secs);
+
+  const [mintShortBrk, setMintShortBrk] = useState(PERIODS.shortBrk.mins);
+  const [secShortBrk, setSecShortBrk] = useState(PERIODS.shortBrk.secs);
+
+  const [mintLongBrk, setMintLongBrk] = useState(PERIODS.longBrk.mins);
+  const [secLongBrk, setSecLongBrk] = useState(PERIODS.longBrk.secs);
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
+    PERIODS.work.mins = mintWork;
+    PERIODS.work.secs = secWork;
+
+    PERIODS.shortBrk.mins = mintShortBrk;
+    PERIODS.shortBrk.secs = secShortBrk;
+
+    PERIODS.longBrk.mins = mintLongBrk;
+    PERIODS.longBrk.secs = secLongBrk;
+  };
+
+  return (
+    <>
+      <Form onSubmit={handelSubmit}>
+        <Form.Label className="text-center" style={{ width: '100%' }}><h4>Work</h4></Form.Label>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <span>Minutes</span>
+          <span>Seconds</span>
+        </div>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <Form.Control type="number" className="w-25" name="workmint" min="0" defaultValue={mintWork} onChange={(e) => setMintWork(e.target.value)} />
+          <Form.Control type="number" className="w-25" min="0" defaultValue={secWork} onChange={(e) => setSecWork(e.target.value)} />
+        </div>
+
+        <Form.Label className="text-center mt-2" style={{ width: '100%' }}><h4>Short Break</h4></Form.Label>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <span>Minutes</span>
+          <span>Seconds</span>
+        </div>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <Form.Control type="number" className="w-25" min="0" defaultValue={mintShortBrk} onChange={(e) => setMintShortBrk(e.target.value)} />
+          <Form.Control type="number" className="w-25" min="0" defaultValue={secShortBrk} onChange={(e) => setSecShortBrk(e.target.value)} />
+        </div>
+
+        <Form.Label className="text-center mt-2" style={{ width: '100%' }}><h4>Long Break</h4></Form.Label>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <span>Minutes</span>
+          <span>Seconds</span>
+        </div>
+        <div className="d-flex flex-row justify-content-sm-around">
+          <Form.Control type="number" className="w-25" min="0" defaultValue={mintLongBrk} onChange={(e) => setMintLongBrk(e.target.value)} />
+          <Form.Control type="number" className="w-25" min="0" defaultValue={secLongBrk} onChange={(e) => setSecLongBrk(e.target.value)} />
+        </div>
+        <div className=" d-flex flex-row justify-content-center">
+          <Button variant="outline-dark" type="submit" className="w-25 mt-5">Save</Button>
+        </div>
+      </Form>
+    </>
+  );
+};
```

Comment:
You have a lot of repeated code here, can you think of a way to reduce the repetition using `map`? Here's an idea:

```suggestion
  const [periods, setPeriods] = useState(PERIODS);

  const handleSubmit = (e) => {
    e.preventDefault();
    Object.assign(PERIODS, periods);
  }
  
  const changePeriod = (periodName, propertyKey, propertyValue) => {
    setPeriods({
      ...periods,
      [periodName]: {
        ...periods[periodName],
        [propertyKey]: propertyValue,
      }
    });
  }
  
  return (
    <Form>
      {periods.map((period, periodName) => (
        <h4 className="text-center">Work</h4>
        <div className="d-flex flex-row justify-content-sm-around">
          <Form.Label for={`${period.id}-mins`}>Minutes</Form.Label>
          <Form.Label for={`${period.id}-secs`}>Seconds</Form.Label>
        </div>
        <div className="d-flex flex-row justify-content-sm-around">
          <Form.Control
            type="number"
            className="w-25"
            name={`${period.id}-mins`}
            min="0"
            defaultValue={period.mins}
            onChange={(e) => changePeriod(periodName, 'mins', e.target.value)}
          />
          <Form.Control
            type="number"
            className="w-25"
            name={`${period.id}-secs`}
            min="0"
            defaultValue={period.secs}
            onChange={(e) => changePeriod(periodName, 'secs', e.target.value)}
          />
        </div>
      ))}
    </Form>
  )
```